src/gocam/cli.py

```diff
@@ -476,5 +480,273 @@ def flatten_models(input_file, input_format, output_format, output_file, fields)
         click.echo(output_content)
 
 
+@cli.command("translate-collection")
+@click.option(
+    "--url",
+    default="http://current.geneontology.org/products/json/noctua-models-json.tgz",
+    help="URL to download the GO-CAM models tarball",
+    show_default=True,
+)
+@click.option(
+    "--formats",
+    multiple=True,
+    type=click.Choice(["networkx", "cx2"]),
+    default=["networkx", "cx2"],
+    help="Formats to translate models to (can specify multiple)",
+    show_default=True,
+)
+@click.option(
+    "--output-networkx",
+    default="/tmp/networkx",
+    help="Output directory for NetworkX format files",
+    show_default=True,
+)
+@click.option(
+    "--output-cx2",
+    default="/tmp/cx2",
+    help="Output directory for CX2 format files",
+    show_default=True,
+)
```

Comment:
Minor: Consider using the `click.Path` type here to simplify checking if a writable directory is provided. It can also be configured it to return a `pathlib.Path` instead of a `str`. Then you could probably replace a lot of the low-level `os.path` function calls with `Path` method calls.